bin/generate.js

```diff
@@ -0,0 +1,31 @@
+const Handlebars = require('handlebars');
+const fs = require('fs');
+const events = require(__dirname + '/../events.json');
+const template = fs.readFileSync(__dirname + '/../template.handlebars',  "utf-8");
+
+function render (source, events, url) {
```

Comment:
Can you add documentation to this function? What do `source`, `events` and `url` mean in this case.